### ISSUE TITLE
idk why is not working Transcribe Data

### ISSUE THREAD

**Issue by f0lk-amy** [SWivid/F5-TTS#685]
### Checks

- [X] This template is only for usage issues encountered.
- [X] I have thoroughly reviewed the project documentation but couldn't find information to solve my problem.
- [X] I have searched for existing issues, including closed ones, and couldn't find a solution.
- [X] I confirm that I am using English to submit this report in order to facilitate communication.

### Environment Details

i watched  Jarods Journey tutorial  

### Steps to Reproduce

in Transcribe Data 
Device set to use cpu --(i dont know why it says cpu if i have a gpu)
C:\Users\Eduard Stefan\Desktop\deleteme\F5-TTS\venv\Lib\site-packages\transformers\models\whisper\generation_whisper.py:512: FutureWarning: The input name `inputs` is deprecated. Please make sure to use `input_features` instead.
  warnings.warn(
You have passed task=transcribe, but also have set `forced_decoder_ids` to [[1, None], [2, 50360]] which creates a conflict. `forced_decoder_ids` will be ignored in favor of task=transcribe.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

but when i followed Jarods Journey it didn't work but i used # git submodule update --init --recursive  and it worked after that
